storybook/src/components/legend-item/legend-item.content.ts

```diff
@@ -0,0 +1,16 @@
+import { html } from "lit-html";
+
+export const bodyWithSelectables = html`<fieldset>
```

Comment:
In deze twee consts dupliceren we markup en maken we geen gebruik van de data driven template models (`component.models.ts`).

Dit probleem lossen we op twee manieren op:
* De consts omzetten naar functions. De template container wordt vervolgens doorgegeven. Bijvoorbeeld `storybook/src/components/modal/modal.stories.ts`.
* De benodigde Lit templates in de Story template factory maken. Bijvoorbeeld `storybook/src/components/highlight-box/highlight-box.stories.ts`

Door de component templates uit de template container te halen is er één bron van waarheid voor de juiste markup en is onze aanroep static typed.